examples/with-mongodb/pages/api/pets/index.js

```diff
@@ -0,0 +1,32 @@
+import dbConnect from '../../../utils/dbConnect'
+import Pet from '../../../models/Pet'
+
+dbConnect()
```

Comment:
This should be awaited in the handler body.

Reply:
Hi @Timer, thanks for the feedback!! Would writing just `await dbConnect()` in the handler function be correct (like on line 8)? It looks like that functions well, but just want to make sure that I got the best practices down :smile: